app/institutions/dashboard/-components/panel/template.hbs

```diff
@@ -0,0 +1,10 @@
+<Panel local-class='panel' as |panel|>
+    <panel.heading @title='{{this.title}}' />
```

Comment:
Once it's template-only, `this` is no longer defined. Also, no need to use quotes here.
```suggestion
    <panel.heading @title={{@title}} />
```